tutor/resources/styles/book-content/note.scss

```diff
@@ -3,6 +3,7 @@ $feature-title-is-element: '[data-has-label=true][data-label=""]';
 $tutor-note-title-element: '[data-type=title]:first-child';
 
 $tutor-note-margin-horizontal: 38px;
+$tutor-note-margin-vertical: 71px;
 %tutor-empty-rules { }
```

Comment:
These units are really strange to me.  why are they so precise?

Reply:
eh, just picked something that looked good